models/spaceid/bnb/spaceid_bnb_schema.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+
+models:
+  - name: spaceid_bnb_view_registrations
```

Comment:
We can simply name the model as `spaceid_bnb_registrations`, also we need to change the alias.

Reply:
Okay. Renamed.